src/duel.cpp

```diff
@@ -411,7 +411,7 @@ void DUEL_TimeExpired( void )
 			}
 			else
 			{
-				SERVERCOMMANDS_PrintHUDMessageFadeOut( szString, 160.4f, 75.0f, 320, 200, CR_RED, 3.0f, 2.0f, "BigFont", false, MAKE_ID('C','N','T','R') );
+				SERVERCOMMANDS_PrintHUDMessage( szString, 160.4f, 75.0f, 320, 200, HUDMESSAGETYPE_FADEOUT, CR_RED, 3.0f, NULL, 2.0f, "BigFont", MAKE_ID( 'C', 'N', 'T', 'R' ) );
```

Comment:
The `NULL` is the `float fInTime` argument, right? Then, I'd put `0.0f` instead. Same applies to all other occasions. `NULL` seems to imply that you want to pass a NULL pointer.

Reply:
I will change that (I thought by using `NULL` it would better convey that this argument wasn't being used, sorry for the confusion), thank you for correcting me on this.

Reply:
No problem and thanks for changing!